apps/segment/segment.js

```diff
@@ -1172,7 +1191,7 @@ async function showInfo() {
             document.getElementById('removeModel'+modelCount).addEventListener('click', () => {
               deleteModel(name);
             });
-            modelCount+=1;
+           modelCount+=1;
```

Comment:
this should fix the style build
```suggestion
            modelCount+=1;
```

Reply:
Ya thanks.  I have done it.